apps/studio/components/interfaces/Support/SupportFormV2.tsx

```diff
@@ -0,0 +1,696 @@
+      }
+    }
+
+    submitSupportTicket(payload)
+  }
+
+  useEffect(() => {
+    if (subject !== urlSubject && subject.trim().length > 0) {
+      handleDocsSearchDebounced(subject.trim())
+    } else {
+      setDocsResults([])
+    }
+  }, [subject, urlSubject])
+
+  useEffect(() => {
+    if (subject.trim().length > 0 && searchState.status === 'fullResults') {
+      setDocsResults(searchState.results)
+    } else if (searchState.status === 'noResults' || searchState.status === 'error') {
+      setDocsResults([])
+    }
+  }, [searchState])
+
+  useEffect(() => {
+    if (!uploadedFiles) return
+    const objectUrls = uploadedFiles.map((file) => URL.createObjectURL(file))
+    setUploadedDataUrls(objectUrls)
+
+    return () => {
+      objectUrls.forEach((url: any) => URL.revokeObjectURL(url))
+    }
+  }, [uploadedFiles])
+
+  useEffect(() => {
+    // For prefilling form fields via URL, project ref will taking higher precedence than org slug
+    if (isSuccessOrganizations && isSuccessProjects) {
+      if (ref) {
+        const selectedProject = allProjects.find((p) => p.ref === ref)
+        if (selectedProject !== undefined) {
+          form.setValue('organizationSlug', selectedProject.organization_slug)
+          form.setValue('projectRef', selectedProject.ref)
+        }
+      } else if (slug) {
+        const selectedOrganization = organizations?.find((org) => org.slug === slug)
+        if (selectedOrganization !== undefined) {
+          form.setValue('organizationSlug', selectedOrganization.slug)
+        }
+      } else if (ref === undefined && slug === undefined) {
+        form.setValue('organizationSlug', organizations[0].slug)
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [ref, slug, isSuccessOrganizations, isSuccessProjects])
+
+  useEffect(() => {
+    if (urlCategory) {
+      const validCategory = CATEGORY_OPTIONS.find((option) => {
+        if (option.value.toLowerCase() === ((urlCategory as string) ?? '').toLowerCase())
+          return option
+      })
+      if (validCategory !== undefined) form.setValue('category', validCategory.value)
+    }
+  }, [urlCategory])
+
+  useEffect(() => {
+    if (urlSubject) form.setValue('subject', urlSubject)
+  }, [urlSubject])
+
+  useEffect(() => {
+    if (urlMessage) form.setValue('message', urlMessage)
+  }, [urlMessage])
+
+  return (
+    <Form_Shadcn_ {...form}>
+      <form
+        id="support-form"
+        className={cn('flex flex-col gap-y-8 max-w-[620px]')}
+        onSubmit={form.handleSubmit(onSubmit)}
+      >
+        <h3 className="text-xl px-6">How can we help?</h3>
+
+        <FormField_Shadcn_
+          name="organizationSlug"
+          control={form.control}
+          render={({ field }) => (
+            <FormItemLayout
+              className="px-6"
+              layout="vertical"
+              label="Which organization is affected?"
+            >
+              <FormControl_Shadcn_>
```

Comment:
FormControl should be inside the Trigger I think

Reply:
ahh yeah the form field seems to be working with this set up, tested locally and on preview - i'll mark as resolved then but just let me know if I should change this!

Reply:
cause i see what you mean with this example here
https://github.com/supabase/supabase/blob/master/apps/studio/pages/new/%5Bslug%5D.tsx#L493